src/UserSpaceInstrumentation/InstrumentProcess.cpp

```diff
@@ -153,7 +154,7 @@ class InstrumentedProcess {
 ErrorMessageOr<std::unique_ptr<InstrumentedProcess>> InstrumentedProcess::Create(
     const CaptureOptions& capture_options) {
   std::unique_ptr<InstrumentedProcess> process(new InstrumentedProcess());
-  const pid_t pid = capture_options.pid();
+  const pid_t pid = orbit_base::ToNativeProcessId(capture_options.pid());
```

Comment:
I wish we passed pid and a list of instrumented functions here instead of capture_options. In which case the conversion could have been done on the caller side.

Reply:
yeah...